### PR TITLE
feat: `homey app manage` to open devtools app page

### PR DESCRIPTION
**Pull request by tjallingt**
now the url commands are: 

```
homey docs
homey tools
homey app view
homey app manage
```

### CODE REVIEW

File: bin/cmds/app/manage.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const colors = require('colors');
+const open = require('open');
+const { Log } = require('../../..');
+const { App } = require('../../..');
+
+exports.desc = 'View your app in the Homey Apps Store';
```
Comment (mrlukasbos) [inline bin/cmds/app/manage.js:8]: This description is not correct

**Review by mrlukasbos — CHANGES_REQUESTED** [review]
Why remove the config imports?

**Comment by tjallingt** [comment]
I mostly removed it since it was used inconsistently and only in one place. Now all the "open commands" are more similar/simpler.

File: bin/cmds/app/manage.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const colors = require('colors');
+const open = require('open');
+const { Log } = require('../../..');
+const { App } = require('../../..');
+
+exports.desc = 'View your app in the Homey Dev Tools';
```
Comment (mrlukasbos) [inline bin/cmds/app/manage.js:8]:
```suggestion
exports.desc = 'View your app in the Homey Developer Tools';
```

**Review by mrlukasbos — APPROVED** [review]

**Pull request merged**